assets/sass/components/global/_googlesitekit-entity-search.scss

```diff
@@ -42,7 +42,17 @@
 		.googlesitekit-entity-search__close {
 			height: 32px;
 			min-width: 36px;
-			padding: 0;
+
+			/**
+			*	We need to set 5px left and right padding for
+			*	larger screens since we have set the close button
+			*	icon size 20 for consistency.
+			*/
+			padding: 0 5px;
```

Comment:
@hussain-t The way you have it makes a harder to read and understand. Basically you want padding set to `0` on small screens only which can be a bit confusing when reading the code, because you set  `padding: 0 5px` and then you set it to `0` for mobile. In the mobile first approach, it's better to have the following:
```css
padding: 0;

@media (min-width: $bp-tablet) {
     padding: 0 5px;
}
```
It's easier to read and understand 😁 This may be redundant if we set the dimensions of the icon to `30` but worth noting for the future 👍